app/controllers/submissions_controller.rb

```diff
@@ -1,11 +1,16 @@
 class SubmissionsController < ApplicationController
   def new
     @submission = Submission.new
+    @submission.submission_packets.build.build_poem
   end
 
   def create
-    @submission = Submission.create(submission_params_with_user)
+    @submission = current_user.submissions.build(submission_params)
+    @submission.submission_packets.map do |packet|
+      packet.poem.user_id = current_user.id
```

Comment:
I thought so too, but I'm getting the database complaining that I'm trying to create a record in  `poems` with no `user_id` value.

Possible I'm missing something somewhat obvious. If you've got a few minutes to pair and spotcheck, that'd be great.